packages/create-sitecore-jss/scripts/post-install.ts

```diff
@@ -0,0 +1,12 @@
+﻿const execSync = require('child_process').execSync;
+
+// restore dependencies added to yarn.lock file post initializing a sample app using the watch script.
```

Comment:
This works if you are _only developing templates_, but creates a bit of a reverse issue when you are _also developing packages_. In this case, you may revert legitimate yarn.lock changes (e.g. you `yarn add` a new dependency to `sitecore-jss` package). 

Maybe we make this another `watch.json` config option to allow you to toggle it on/off depending on what you're working on?

Reply:
Are we talking about adding a dependency while initializing an app using a watch script?

Reply:
I was thinking (1) adding a dependency (working on base package), and then (2) you go to work on something in an initializer and do a yarn watch. Your legit yarn lock change from 1 would be wiped out.